src/bundle/Controller/AuditExportController.php

```diff
@@ -74,12 +74,16 @@ public function __construct(
      */
     public function exportAction(Request $request): Response
     {
+        $repository = $this->container->get('ezpublish.api.repository');
         $this->permissionAccess('uiaudit', 'export');
-
-        $limit = $request->get('limit', 10);
-        $page = $request->get('page', 1);
-
-        $query = $this->auditService->buildExportQuery();
+        $limit        = $request->get('limit', 10);
+        $page         = $request->get('page', 1);
+        $hasAllAccess = $repository->hasAccess('uiaudit', 'configure', $repository->getCurrentUser());
```

Comment:
@deprecated since 6.6, to be removed. Use PermissionResolver::hasAccess() instead.